worker/worker/ffmegtools.go

```diff
@@ -14,12 +16,42 @@ func getDuration(file string) (float64, error) {
 	return gjson.Get(probe, "format.duration").Float(), nil
 }
 
-func getCodec(file string) (string, error) {
+func getVideoCodec(file string) (string, error) {
 	probe, err := probe(file)
 	if err != nil {
 		return "", err
 	}
-	return gjson.Get(probe, "streams.0.codec_name").String(), nil
+	codecNumber := gjson.Get(probe, "streams.#").Int()
```

Comment:
I think a more descriptive name would be nice, something like 
```suggestion
	nStreams := gjson.Get(probe, "streams.#").Int()
```